spec/proof_hash_scheme.adoc

```diff
@@ -0,0 +1,17 @@
+= Proof Hash Scheme
+:cddl: ./cddl/
+
+The following list of hash schemes is ordered by attribute id
+
+== 0
+
+Each node contains a "kv hash", which is the hash of a prefix concatenanted with the nodes key and its value.
+The hash of the node is the hash of a distinct prefix, the kv hash, and the hash of each of the left and right child nodes.
+
+```
+kv_hash = H(0x00, key, value)
+node_hash = H(0x01, kv_hash, left_child_hash, right_child_hash)
```

Comment:
Same as above.

Reply:
Actually, the node hash does _not_ have the lengths in the hash.